internal/provider/schema_cloudflare_email_routing_catch_all.go

```diff
@@ -0,0 +1,70 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceCloudflareEmailRoutingCatchAllSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"zone_id": {
+			Description: "The zone identifier to target for the resource.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"tag": {
+			Description: "Routing rule identifier.",
+			Type:        schema.TypeString,
+			Computed:    true,
+		},
+		"name": {
+			Description: "Routing rule name.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"enabled": {
+			Description: "Routing rule status.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+		},
+		"matcher": {
+			Description: "Matching patterns to forward to your actions.",
+			Type:        schema.TypeSet,
+			Required:    true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"type": {
+						Description:  "Type of matcher.",
```

Comment:
```suggestion
						Description:  fmt.Sprintf("Type of matcher. %s", renderAvailableDocumentationValuesStringSlice([]string{"all"})),
```

(don't forget to update your imports for `fmt`)